FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -810,6 +810,28 @@ def getComponentCfg(self, componentType, system, component, compInstance, extens
           loadCfg.loadFromFile(cfgTemplatePath)
           compCfg = loadCfg.mergeWith(compCfg)
 
+        # Path to the root dirac.cfg
+        cfgTemplatePath = os.path.join(rootPath, 'dirac.cfg')
+        # Read it and extract the interesting part
+        if os.path.exists(cfgTemplatePath):
+          gLogger.notice('Extracting default configuration from', cfgTemplatePath)
+          with open(cfgTemplatePath, 'r') as diracCfgFile:
+            diracCfgContent = diracCfgFile.read()
+            # We take all what is between the BEGIN and END marker of this component
+            componentRegex = r'### BEGIN %s\n(.*?)### END'%componentModule
```

Comment:
This gives, e.g.
```
   TransformationCleaningAgent
   {...}
```
But what is needed is, e.g.
```
Agents
{
   TransformationCleaningAgent
   {...}
}
```
Why not just load all of dirac.cfg and look for Systems/<System>/[Agents|Services|Executors]/componentModule